cursorless-talon/src/spoken_forms_output.py

```diff
@@ -0,0 +1,46 @@
+import json
+from pathlib import Path
+from typing import TypedDict
+
+from talon import app
+
+from .command import CursorlessCommand
+
+SPOKEN_FORMS_OUTPUT_PATH = Path.home() / ".cursorless" / "spokenForms.json"
```

Comment:
Are we sure about this path? Tough to change. Should we name the `.json` something more generic, in case we want to put things other than spoken forms there? Not sure we'll ever want more than one json, but possibly?

Reply:
go with `state.json` and use `spokenForms` as the key